var/spack/repos/builtin/packages/arborx/package.py

```diff
@@ -9,42 +9,61 @@
 class Arborx(CMakePackage):
     """ArborX is a performance-portable library for geometric search"""
 
-    homepage = "http://github.com/arborx/arborx"
-    url      = "https://github.com/arborx/arborx/archive/v0.8-beta2.tar.gz"
+    homepage = "https://github.com/arborx/arborx"
+    url      = "https://github.com/arborx/arborx/archive/v0.9-beta.tar.gz"
     git      = "https://github.com/arborx/arborx.git"
 
     version('master', branch='master')
-    version('0.8-beta2', sha256='e68733bc77fbb84313f3ff059f746fa79ab2ffe24a0a391126eefa47ec4fd2df')
+    version('0.9-beta', sha256='b349b5708d1aa00e8c20c209ac75dc2d164ff9bf1b85adb5437346d194ba6c0d')
+
+    # ArborX relies on Kokkos to provide devices, providing one-to-one matching
+    # variants. The only way to disable those devices is to make sure Kokkos
+    # does not provide them.
+    kokkos_backends = {
+        'serial': (True,  "enable Serial backend (default)"),
+        'cuda': (False,  "enable Cuda backend"),
+        'openmp': (False,  "enable OpenMP backend"),
+        'hip': (False,  "enable HIP backend")
+    }
 
-    variant('cuda', default=False, description='enable Cuda backend')
-    variant('openmp', default=False, description='enable OpenMP backend')
-    variant('serial', default=True, description='enable Serial backend (default)')
     variant('mpi', default=True, description='enable MPI')
+    for backend in kokkos_backends:
+        deflt, descr = kokkos_backends[backend]
+        variant(backend.lower(), default=deflt, description=descr)
```

Comment:
You could also use a single multi-valued variant here, not sure what your preference is but it doesn't matter to me

Reply:
Lets leave it like this for now to be consistent with `kokkos-kernels`.